apps/arweave/src/ar_mining_worker.erl

```diff
@@ -601,6 +545,144 @@ handle_task({compute_h2_for_peer, Candidate, _ExtraArgs}, State) ->
 %%% Private functions.
 %%%===================================================================
 
+process_chunks(WhichChunk, Candidate, RangeStart, ChunkOffsets, State) ->
+	PackingDifficulty = Candidate#mining_candidate.packing_difficulty,
+	SubchunksPerRecallRange = ar_block:get_max_nonce(PackingDifficulty),
```

Comment:
```suggestion
	SubChunksPerRecallRange = ar_block:get_max_nonce(PackingDifficulty),
```

ditto for all `Subchunks`

Reply:
Also in this case I do think "NoncesPerRecallRange" or something like that might be better. In spora_2_6 there's one nonce per chunk in replica.2.9 there's one nonce per subchunk - so "SubChunksPerRecallRange" isn't totally accurate for spora_2_6 mining, but NoncesPerRecallRange is